src/builtin-addons/core/intl-definition-provider.ts

```diff
@@ -0,0 +1,34 @@
+import { Definition } from 'vscode-languageserver-types';
+import { DefinitionFunctionParams, Server } from '../..';
+import { isLocalizationHelperTranslataionName } from '../../utils/ast-helpers';
+import { getTranslations } from './intl-utils';
+
+export default class IntlDefinitionProvider {
+  server: Server;
+
+  async onInit(server: Server) {
+    this.server = server;
+  }
+
+  async onDefinition(root: string, params: DefinitionFunctionParams): Promise<Definition | null> {
```

Comment:
should return type to be Promise<Definition[]> ? (we use results, and it's always array)

Reply:
I just reused types from `script-definition-provider` and `template-definition-provider`. Should we change them also there?

Reply:
sure, let's do it! :)

Reply:
no, don't think we need to change it here, let's just have proper typings here to be able to catch some errors if we will change API